src/routes/safe_apps/tests/routes.rs

```diff
@@ -20,7 +22,7 @@ async fn safe_apps() {
     let mut mock_http_client = MockHttpClient::new();
 
     let safe_apps_request = Request::new(config_uri!(
-        "/v1/safe-apps/?chainId={}&clientUrl={}",
+        "/v1/safe-apps/?chainId={}&clientUrl={}&url=",
```

Comment:
What's the purpose of adding url here if you are not using it?

Reply:
I do not have a strong reasoning, but I think that it always gets added to config service request with an empty value even if not used: https://github.com/safe-global/safe-client-gateway/blob/248a9a1bf8b45fb8d6bcb80a5ef22d5fd78768cb/src/routes/safe_apps/handlers.rs#L15

Also, it's used the same way in other tests for the `client_url` param

Reply:
Makes sense then, thanks 😉